app/routes/application.js

```diff
@@ -1,8 +1,11 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  actions: {
+  activate: function() {
+    this.get('session').fetch();
+  },
 
+  actions: {
     signInViaGithub: function() {
       var route = this;
 
```

Comment:
do we need the code here to go get the session and initiate the oauth?

something like:

this.get('session').open('github-oauth2').then(function(session){
 then we put a redirect in here later